websites/docs-smoke-test/src/content/components/cards.mdx

```diff
@@ -0,0 +1,203 @@
+   * Icons are supported anywhere but only those that are officially provided by the kit. An icon is sized into a fixed 48x48px square.
+ * Cards can occur in two interaction variants, "active" (visually elevated) and "flat" (visually an outline only).
+ * Cards must be embedded into a wrapper component that defines the responsive sizes (two aside / three aside) and parameters that must be identical in a give row or grid (active card area, the heading size)
+
+# "Cards" Wrapper
+
+ Properties (for authors):
+ * `active` (boolean): if set, the whole area of all child cards is a link, the card is elevated and has a hover effect if a link url is set.
+ * `narrow` (boolean): by default, two cards are shown side by side (except mobile viewports). if `narrow` is passed, the minimum card width is reduced so that three cards fit the content width on microsite landing pages.
+ * `smallTitle` (boolean): lets all cards have a smaller
+
+## Minimal form
+
+```jsx
+<Cards>
+  <Card>foo</Card>
+  <Card>bar</Card>
+  <Card>next row unless three aside (narrow) prop passed?</Card>
+  <Card>next row in any case</Card>
+</Cards>
+```
+### Output
+
+<Cards>
+  <Card>foo</Card>
+  <Card>bar</Card>
+  <Card>next row unless three aside (narrow) prop passed?</Card>
+  <Card>next row in any case</Card>
+</Cards>
+
+
+## All options configured
+
+```jsx
+<Cards active narrow smallTitle>
+  <Card>foo</Card>
+  <Card>bar</Card>
+  <Card>next row unless three aside (narrow) prop passed?</Card>
+  <Card>next row in any case</Card>
+</Cards>
+```
+
+### Output
+
+<Cards active narrow smallTitle>
+  <Card>foo</Card>
+  <Card>bar</Card>
+  <Card>next row unless three aside (narrow) prop passed?</Card>
+  <Card>next row in any case</Card>
+</Cards>
+
+
+## Pseudo-CSS:
+
+This is for illustration. The implementation can be done differently. The design intent matches the CSS flexbox intent, but CSS grid might be more powerful although not originally being intended for rows with floating overflow.
+
+```css
+.cards {
+	width: 100%;
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    align-items: stretch;
+ }
+.cards > * {
+	flex-basis: 332px; /* the minimum width to never be three in a row */
+	flex-grow: 1;
+	flex-shrink: 1;
+	/* No maximum width possible here in flexbox, i.e. there could be one card taking full width (not intended, but acceptable)  */
+}
+.cards.narrow > * {
+	flex-basis: 250px; /* the minimum width to never be four in a row */
+}
+```
+
+# "Card": the actual card
+
+Properties (for authors):
+ * `title` (string): the card title, no markdown allowed.
+ * `href` (string): the link target of the bottom link and, if `active` is set, the whole card. `href` has no effect when neither `active` or `link` are set.
+ * `link` (string): the bottom link text. If not set, no bottom link is rendered at all.
```

Comment:
The name is confusing with an actual link. Maybe `linkLabel` or something?

```suggestion
 * `linkLabel` (string): the bottom link text. If not set, no bottom link is rendered at all.
```

Reply:
I was going back and forth on that question and was not sure.  I was not happy with "linkLabel" because setting it does not only change the label but also trigger that there's a link at all.  But just "link" is definitely likely to be confused with the href. 

Needs to be decided in conjunction with the "active" prop (see above).   

if we like it more explicit we can split it into four parameters and stay away from abstractions like "active":  

 * `href` (string) - the url, pointless if neither clickable nor withLink
 * `clickable`  (boolean, on wrapper)
 * `withLink` (boolean) -> use "Read more..."  as the linkText by default. 
 * `linkText` (string)

Reply:
PS: I now renamed "active" to "clickable" and "link"  to "textLink".   I don't think that still requires an extra "withLink". 

OK?